src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java

```diff
@@ -48,6 +49,7 @@
 public class AudioController implements AudioApi {
 
     private final AudioService audioService;
+    private final AudioTransformationService audioTransformationService;
```

Comment:
I'm not sure why this dependency was added - it doesn't look to be used here?

Reply:
good spot, i had added a temporary endpoint in here while local testing, but forgot to remove this.
removed now.